style/style.css

```diff
@@ -0,0 +1,312 @@
+
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: "Roboto", sans-serif;
+}
+
+body{
+    background-color: #ffffff;
+    display: grid;
+    grid-template-columns: repeat(1, 1fr);
+    grid-template-rows: 200px auto auto;
+    grid-template-areas: 
+    "hd"
+    "section"
+    "footer";
+    gap: 1rem;
+}
+
+
+.header{
+    /* background-color: red; */
+    grid-area: hd;
+    background-image: url("/img/header\ pic.avif");
+    object-fit: cover;
```

Comment:
`object-fit` is for `img` tags; you've used a background image here so I don't think you need this property. Try unticking it in devtools Styles panel and see if the layout changes.